packages/components/src/theme/ComposerTheme/Radius/radius.ts

```diff
@@ -10,4 +10,5 @@ export const defaultRadius = {
   'radii-modal': '0.5rem',
   'radii-alertdialog': '0.5rem',
   'radii-popover': '0.5rem',
+  'radii-button': '0.375rem',
```

Comment:
you have to use this value in the button theming

Reply:
@tkanzakic Oops, forgot that
cc: @guilean